.gitignore

```diff
@@ -7,3 +7,4 @@ coverage/
 rdoc/
 .sass-cache/
 Gemfile.lock
+config.yml
```

Comment:
@SilasReinagel that's a bad idea, it will ignore it during deployment to Heroku, see `.rultor.yml`

Reply:
So many hidden interactions. Okay, I will remove it.